source/views/sis/course-search/detail/index.js

```diff
@@ -44,11 +44,21 @@ function Information({course}: {course: CourseType}) {
 	const credits = course.credits ? (
 		<Cell cellStyle="LeftDetail" detail="Credits" title={course.credits} />
 	) : null
+	const type = <Cell cellStyle="LeftDetail" detail="Type" title={course.type} />
+	const passFail = course.pn ? 'Yes' : 'No'
+	const pn = <Cell cellStyle="LeftDetail" detail="Pass/Fail" title={passFail} />
+	const prerequisites = course.prerequisites ?
+		course.prerequisites.replace("Prerequisite: ", "") : 'None'
```

Comment:
Are these starting with "Prerequisites"? I can remove that in the source.

Reply:
Yeah they are! That would be dope.